content/en/glossary/terms/apm.md

```diff
@@ -0,0 +1,12 @@
+---
+title: APM
+related_terms:
+    - Tracer
```

Comment:
hey @kayayarai - Tracer is a related term here, so there is a link to `#tracer`, but it doesn't seem like there is a definition for tracer yet.

Reply:
oh, let's just take this one out for now then
```suggestion
```